processing/bloom_handler.go

```diff
@@ -156,7 +169,15 @@ func MakeBloomHandlerFromFile(bloomFilename string, compressed bool,
 			return nil, err
 		}
 	}
+	log.WithFields(log.Fields{
+		"domain": "bloom",
+	}).Infof("loading Bloom filter '%s'", bloomFilename)
 	bh := MakeBloomHandler(iocBloom, databaseChan, forwardHandler, alertPrefix)
+	err = checkBlacklisted(iocBloom, blacklistIOCs)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
You may want to use `if err := checkBlacklisted(...); err != nil { ... }` ... and move this block up before initializing `bh`.

Reply:
True (moving up). But IMHO the way the code is written (assignment and check on two lines) is more idiomatic looking that the code of the rest of the project.